tracer/test/test-applications/regression/AspNetCoreSmokeTest/AspNetCoreSmokeTest.csproj

```diff
@@ -1,12 +1,22 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1;netcoreapp3.0;netcoreapp3.1;net5.0;net6.0;net7.0;net8.0;net9.0</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net462;$(TargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks>net6.0;net7.0;net8.0;net9.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(PublishTrimmed)' != 'true'">netcoreapp2.1;netcoreapp3.0;netcoreapp3.1;net5.0;$(TargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks Condition="'$(PublishTrimmed)' != 'true' AND '$(OS)' == 'Windows_NT'">net462;$(TargetFrameworks)</TargetFrameworks>
     <CheckEolTargetFramework>false</CheckEolTargetFramework>
     <LangVersion>latest</LangVersion>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(PublishTrimmed)' == 'true'">
+    <PublishTrimmed>true</PublishTrimmed>
```

Comment:
not a blocker, but this line seems redundant
"if `PublishTrimmed` is true, set `PublishTrimmed` to true"

Reply:
Yeah, it probably is 😅 I just copy pasted from our existing trimmed sample, and then set `PublishTrimmed` as an env var. I'm not _100%_ whether these _are_ the same things or not 🤔

Reply:
Will try it anyway

```suggestion
```

Reply:
Env vars with valid msbuild property names are read into msbuild as build properties, so there's no real distinction here: both `Condition="'$(PublishTrimmed)' == 'true'"` and `<PublishTrimmed>true</PublishTrimmed>` are referencing the same property.

https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-properties#environment-properties